### ISSUE TITLE
Preparing cookiecutter message not themed

### ISSUE THREAD

**Issue by zooba** [Microsoft/PTVS#2900]
![image](https://user-images.githubusercontent.com/1693688/29195226-2e302c22-7de2-11e7-9a77-b0f871bb06c8.png)

Should be white. Also, the loading image does not spin.
